apis/exoscale/v1/dbaas_exoscale_kafka.go

```diff
@@ -32,54 +32,54 @@ type ExoscaleKafka struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
 	// Spec defines the desired state of a ExoscaleKafka.
-	Spec ExoscaleKafkaSpec `json:"spec,omitempty"`
+	Spec KafkaSpec `json:"spec,omitempty"`
 	// Status reflects the observed state of a ExoscaleKafka.
-	Status ExoscaleKafkaStatus `json:"status,omitempty"`
+	Status KafkaStatus `json:"status,omitempty"`
 }
 
-type ExoscaleKafkaSpec struct {
+type KafkaSpec struct {
 	// Parameters are the configurable fields of a ExoscaleKafka.
-	Parameters ExoscaleKafkaParameters `json:"parameters,omitempty"`
+	Parameters KafkaParameters `json:"parameters,omitempty"`
 }
 
-type ExoscaleKafkaParameters struct {
+type KafkaParameters struct {
 
 	// Service contains Exoscale Kafka DBaaS specific properties
-	Service ExoscaleKafkaServiceSpec `json:"service,omitempty"`
+	Service KafkaServiceSpec `json:"service,omitempty"`
 
 	// Maintenance contains settings to control the maintenance of an instance.
 	Maintenance ExoscaleDBaaSMaintenanceScheduleSpec `json:"maintenance,omitempty"`
 
 	// Size contains settings to control the sizing of a service.
-	Size ExoscaleKafkaDBaaSSizeSpec `json:"size,omitempty"`
+	Size KafkaDBaaSSizeSpec `json:"size,omitempty"`
 
 	// Network contains any network related settings.
 	Network ExoscaleDBaaSNetworkSpec `json:"network,omitempty"`
 }
 
-type ExoscaleKafkaDBaaSSizeSpec struct {
+type KafkaDBaaSSizeSpec struct {
 	// +kubebuilder:default="startup-2"
 
 	// Plan is the name of the resource plan that defines the compute resources.
 	Plan string `json:"plan,omitempty"`
 }
 
-type ExoscaleKafkaServiceSpec struct {
+type KafkaServiceSpec struct {
 	ExoscaleDBaaSServiceSpec `json:",inline"`
 	// KafkaSettings contains additional Kafka settings.
 	KafkaSettings runtime.RawExtension `json:"kafkaSettings,omitempty"`
 
 	// +kubebuilder:validation:Enum="3.2"
 
-	// Version contains the minor version for Kafka.
+	// Version contains the minor version for ExoscaleKafka.
```

Comment:
Nitpick: That's not really an ExoscaleKafka version but really just the Kafka version. Probably some find replace issue?

Reply:
Fixed